src/client/components/description/specification/Specification.js

```diff
@@ -41,13 +42,7 @@ const Specification = (props) => {
 }
 
 Specification.propTypes = {
-  id: PropTypes.number,
-  name: PropTypes.string,
-  rating: PropTypes.number,
-  gengre: PropTypes.string,
-  year: PropTypes.number,
-  duration: PropTypes.number,
-  description: PropTypes.string,
+  movie: PropTypes.object,
```

Comment:
please always try to describe the shape of object, array please see `PropTypes.arrayof(), PropTypes.objectOf(), PropTypes.shape()`

Reply:
Done)